### ISSUE TITLE
The installation failed

### ISSUE THREAD

**Issue by connecteddale** [python/cpython#100024]
# Bug report

<img width="635" alt="image" src="https://user-images.githubusercontent.com/35458521/205707586-42f0152a-fcfb-41a1-ba05-f4ef47ed8141.png">

# Your environment

![image](https://user-images.githubusercontent.com/35458521/205707791-ec4300b3-817d-4c69-957a-7ffdd98e2599.png)

**Comment by ned-deily** [comment]
Thank you for the report. However, we need more information to be able to try to help. Can you say exactly which version of Python you were trying to install and from which URL or web page? Also, if you could try the install again and, if it fails again, please open the Installer Log window (Command-L) and save and attach to this issue the full installer log (or copy and paste it).

**Comment by connecteddale** [comment]
Hello Ned

Yes certainly, it is the latest version on Python for the Mac. The package file is 

python-3.11.0-macos11.pkg downloaded from https://www.python.org/ftp/python/3.11.0/python-3.11.0-macos11.pkg

Here is the log from the installation. I changed my username to username-changed for privacy

Dec  5 22:13:27 MacBook-Pro-2 Installer[3637]: Failed to load specified background image
Dec  5 22:13:41 MacBook-Pro-2 installd[2201]: PackageKit: ----- Begin install -----
Dec  5 22:13:42 MacBook-Pro-2 installd[2201]: PackageKit: Install Failed: Error Domain=PKInstallErrorDomain Code=110 "An error occurred while extracting files from the package “python-3.11.0-macos11.pkg”." UserInfo={NSLocalizedDescription=An error occurred while extracting files from the package “python-3.11.0-macos11.pkg”., NSURL=file:///Users/username-changed/Library/CloudStorage/Dropbox/Mac%20(2)/Downloads/python-3.11.0-macos11.pkg#Python_Framework.pkg, PKInstallPackageIdentifier=org.python.Python.PythonFramework-3.11, NSUnderlyingError=0x14e20df20 {Error Domain=PKXARArchiveErrorDomain Code=101 "archive open failed" UserInfo={NSURL=file:///Users/username-changed/Library/CloudStorage/Dropbox/Mac%20(2)/Downloads/python-3.11.0-macos11.pkg, NSFileOwnerAccountID=501, NSFileHFSTypeCode=0, NSFileSystemFileNumber=36206242, NSFileExtensionHidden=false, NSFileSystemNumber=16777233, NSFileSize=42602603, NSFileGroupOwnerAccountID=20, NSFileHFSCreatorCode=0, NSFileOwnerAccountName=username-changed, NSFilePosixPermissions=420, NSFileCreationDate=2022-12-05 17:25:35 +0000, NSFileType=NSFileTypeRegular, NSFileProtectionKey=NSFileProtectionCompleteUntilFirstUserAuthentication, NSFileGroupOwnerAccountName=staff, NSFileReferenceCount=1, NSFileModificationDate=2022-12-05 17:28:12 +0000, NSLocalizedDescription=archive open failed}}} {
	    NSLocalizedDescription = "An error occurred while extracting files from the package \U201cpython-3.11.0-macos11.pkg\U201d.";
	    NSURL = "file:///Users/username-changed/Library/CloudStorage/Dropbox/Mac%20(2)/Downloads/python-3.11.0-macos11.pkg#Python_Framework.pkg";
	    NSUnderlyingError = "Error Domain=PKXARArchiveErrorDomain Code=101 \"archive open failed\" UserInfo={NSURL=file:///Users/username-changed/Library/CloudStorage/Dropbox/Mac%20(2)/Downloads/python-3.11.0-macos11.pkg, NSFileOwnerAccountID=501, NSFileHFSTypeCode=0, NSFileSystemFileNumber=36206242, NSFileExtensionHidden=false, NSFileSystemNumber=16777233, NSFileSize=42602603, NSFileGroupOwnerAccountID=20, NSFileHFSCreatorCode=0, NSFil
Dec  5 22:13:42 MacBook-Pro-2 Installer[3637]: Install failed: The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.
Dec  5 22:13:42 MacBook-Pro-2 Installer[3637]: Displaying 'Install Failed' UI.
Dec  5 22:13:42 MacBook-Pro-2 Installer[3637]: 'Install Failed' UI displayed message:'The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.'.

**Comment by connecteddale** [comment]
Hello again Ned

What I realised in replying above is that the package is sitting in a downloads folder which is in dropbox. Dropbox have just launched a new version which essentially places the dropbox folder in a disk image. I figured this might be confusing the extraction of the package files - as per the error msg in the log. 

When I moved the package to a local folder it installed fine.

Thank you for your quick reply and thank you for helping me think through the problem. Hopefully this helps someone else who gets stuck with similar.

All the best

Dale
